tensilelite/Tensile/KernelWriter.py

```diff
@@ -4726,6 +4729,9 @@ def _getSingleCodeObjectFile(self, kernel):
 
     subprocess.check_call(args, cwd=self.getAssemblyDirectory())
 
+    if not globalParameters["KeepBuildTmp"]:
+        os.remove(coFileName)
+
     return coFileName
```

Comment:
I am not sure if this is the appropriate place to remove this file, it may be consumed after this function call.  CI is seeing missing .co files, it may be possible the .co's are being deleted before being copied to their final destination:

```
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/.tensile-tox/py3/tmp/popen-gw1/test_config_Tensile_Tests_comm2/1_BenchmarkProblems/Cijk_Ailk_Bjlk_DB_UserArgs_00/00_Final/source/build_tmp/SOURCE/assembly/Cijk_Ailk_Bjlk_DB_UserArgs_MT4x16x16_MI4x4x4_SN_K1_MIWT1_1_SS0.co'
```

Reply:
I will revert this change.